reference/array/functions/array-count-values.xml

```diff
@@ -1,22 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- EN-Revision: 3ce14fde5cd5f15171a7ed2b64fdc7bd652c3842 Maintainer: sobak Status: ready -->
+<!-- EN-Revision: 6b56e6f444e49396ee5122258ee85b99a8543f6a Maintainer: sobak Status: ready -->
 <!-- $Revision$ -->
+<!-- CREDITS: grzesiek -->
 <refentry xml:id="function.array-count-values" xmlns="http://docbook.org/ns/docbook">
  <refnamediv>
   <refname>array_count_values</refname>
-  <refpurpose>Zlicza wszystkie wartości w tablicy</refpurpose>
+  <refpurpose>Zlicza wystąpienia każdej odrębnej wartości w tablicy</refpurpose>
  </refnamediv>
- 
+
  <refsect1 role="description">
   &reftitle.description;
   <methodsynopsis>
    <type>array</type><methodname>array_count_values</methodname>
-   <methodparam><type>array</type><parameter>tablica</parameter></methodparam>
+   <methodparam><type>array</type><parameter>array</parameter></methodparam>
   </methodsynopsis>
   <para>
    <function>array_count_values</function> zwraca tablicę zawierającą
-   wartości <parameter>tablicy</parameter> jako klucze i częstość
-   ich występowania w <parameter>tablicy</parameter> jako wartości.
+   wartości <parameter>array</parameter> (które muszą być &integer; lub &string;) jako klucze i
+   częstość ich występowania w <parameter>array</parameter> jako wartości.
```

Comment:
```suggestion
   ile razy wystąpiły w tablicy <parameter>array</parameter> jako wartości.
```

wydaje mi się że w przeciwnym razie mało kto będzie wiedział, co naprawdę robi ta funkcja, po przeczytaniu "częstość ich występowania".

Oczywiście rozumiem, że to mózg płata figle w momencie tłumaczenia. Osobiście robię to tak, że po przetłumaczeniu zdania (lub czasami przed, jeśli wystarczy szarych komórek :smile:) czytam zdanie w głowie i myślę czy brzmi po polsku.